src/ocl/convolutionocl.cpp

```diff
@@ -749,6 +749,7 @@ static void DirConvFindCore(Handle& handle,
     // Implicit GEMM algo
     if(!use_winograd_only)
     {
+        ConvolutionContext params(xDesc, wDesc, yDesc, conv, conv::Direction::Forward, 0);
```

Comment:
Dead code. Please remove this.

Reply:
Hi @atamazov This code seems related to algorithm-wise selection of solutions, will launch invokers for all solvers under current algorithm. Maybe out of the scope of this PR?

Reply:
Maybe. What this line of code does?

Reply:
@atamazov Yes you are right, I miss understood this line, my fault. This line is removed in updated PR.